e2e/e2e_test.go

```diff
@@ -218,6 +218,40 @@ func TestE2ELocalBackend(t *testing.T) {
 	}
 }
 
+func TestE2EDisabledTask(t *testing.T) {
+	t.Parallel()
+
+	srv, err := newTestConsulServer(t)
+	require.NoError(t, err, "failed to start consul server")
+	defer srv.Stop()
+
+	tempDir := fmt.Sprintf("%s%s", tempDirPrefix, "disabled_task")
+	err = makeTempDir(tempDir)
+	// no defer to delete directory: only delete at end of test if no errors
+	require.NoError(t, err)
+
+	configPath := filepath.Join(tempDir, configFile)
+	err = makeConfig(configPath, disabledTaskConfig(srv.HTTPAddr, tempDir))
+	require.NoError(t, err)
+
+	// deliberately running sync in long running mode. This catches a bug in
+	// when setting buffer_period for a disabled task
```

Comment:
This comment might lose context as it ages, like it's catching an active bug. Do you mind rewording it to be more generic? Just throwing out a suggestion to give express better what I mean

> ... This may catch bugs where functions execute unexpectedly on disabled tasks.

Reply:
Good point! I wrote the comment when it was active and it's definitely lost context since. Let me reword. Thank you!